rholang/src/test/scala/coop/rchain/rholang/interpreter/NormalizeTest.scala

```diff
@@ -1127,6 +1133,7 @@ class ProcMatcherSpec extends FlatSpec with Matchers {
                 1,
                 Send(EVar(BoundVar(0)), List[Par](GInt(47)), false, BitSet(0)),
                 Vector.empty,
+                None,
                 BitSet()
```

Comment:
wouldn't you prefer those AST creations to use named parameters? I'd be fine with such a change, but leave it up to you :)

Reply:
Agreed and done.